src/__tests__/get-page-return/get-page-return.test.tsx

```diff
@@ -40,22 +54,31 @@ describe('getPage() return', () => {
         route: '/page',
       });
       const { nextRoot } = serverRender();
-      const actualNextRoot = document.getElementById('__next');
+      const actualNextRoot = document.getElementById(NEXT_ROOT_ELEMENT_ID);
       expect(nextRoot).toBe(actualNextRoot);
 
       const actualHtml = document.documentElement;
       const { container: expectedHtml } = TLRender(
         <>
-          <head />
+          <head>
+            <meta name="next-head-count" content="0" />
+            <noscript data-n-css=""></noscript>
+          </head>
           <body>
-            <div id="__next">
+            <div id={NEXT_ROOT_ELEMENT_ID}>
               <App Component={Page} />
             </div>
+            <script
+              id="__NEXT_DATA__"
+              type="application/json"
+              dangerouslySetInnerHTML={{
+                __html:
+                  '{"page":"/page","query":{},"buildId":"next-page-tester","props":{}}',
+              }}
+            ></script>
```

Comment:
Should we consider this output (along with `next-head-count` meta, `data-n-css` noscript..) implementation details not depending on our code and keep them out from out expectations?

If Next.js added a new next-tag in a minor version, our test would get red, wouldn't them?